package.json

```diff
@@ -43,6 +43,7 @@
     "flow-babel-webpack-plugin": "1.1.1",
     "fs-extra": "0.30.0",
     "gzip-size": "3.0.0",
+    "handlebars-loader": "^1.7.0",
```

Comment:
Remove, please, `^`

Reply:
Can't remove it.

It is needed for the template processing in the `HtmlWebpackPlugin` webpack plugin of the file `static/index.hbs`.  That is also why it is part of `devDependencies`.

Reply:
I mean:
`"handlebars-loader": "1.7.0",`

Reply:
Oh right.  Fixed the dependency version to just `"1.7.0"`.